configurations/default/env.yml.tmp

```diff
@@ -9,6 +9,10 @@ BUGSNAG_PROJECT_NOTIFIER_API_KEY: project-notifier-api-key
 #BUGSNAG_REPORTING_WINDOW_IN_DAYS: 14
 BUGSNAG_WEBHOOK_PERMITTED_IPS: 104.196.245.109, 104.196.254.247
 
+CONNECTED_DATA_PLATFORM_S3_BUCKET_NAME: bucket-name
+CONNECTED_DATA_PLATFORM_S3_FOLDER_NAME: folder-name
+CONNECTED_DATA_PLATFORM_TRIP_HISTORY_UPLOAD_JOB_DELAY_IN_MINUTES: 5
```

Comment:
Is "delay" the best term here? Isn't it more of a frequency?

Reply:
I've used delay in the past, but agree frequency is better, have updated!